### PR TITLE
Feature/admin page

### PR DESCRIPTION
**Pull request by aliroemhildt**
## Checklist
- [ ] Tests written where required
- [ ] All tests passing
- [ ] Unused code/prys/etc deleted
Testing Coverage at <percentage>%

## Changes Made
- build out structure of admin page
- add initial styling (this can be changed later) 
- made a few changes to app so I could see the admin page (these can all be changed if there are merge conflicts)
- add sample user data file

What (if any) features are you implementing?
- admin page layout (no functionality yet)

What (if anything) did you refactor?
- n/a

Were there any issues that arose?
- n/a

Is there anything that you need from your teammate?
- n/a

Any other comments, questions, or concerns?

### CODE REVIEW

File: src/components/AdminPage/AdminPage.scss [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  height: 250px;
+  width: 250px;
+  margin: 20px;
+  background-color: lightgray;
+
+  &:hover {
+    cursor: pointer;
+    background-color: rgb(189, 189, 189);
+  }
+}
```
Comment (mr-ryan12) [inline src/components/AdminPage/AdminPage.scss:40]: These look so good! Thank you for building this and teaching me some other ways of responsive styling!

**Review by mr-ryan12 — APPROVED** [review]
Thanks again!

**Review by oacortez — APPROVED** [review]
Thank you for creating this branch Ali! I see that our basic admin page is displaying. Thank you for creating fake data as well to display what we need for our product. Everything looks good me.

**Comment by Echory** [comment]
Looks great so far !

**Review by Echory — APPROVED** [review]

**Pull request merged**